api/src/main/java/org/eclipse/microprofile/lra/annotation/Complete.java

```diff
@@ -27,44 +27,132 @@
+ * code. On the other hand, combining it with an <code>&#64;LRA</code> annotation that
+ * begins a new LRA can in certain use cases make sense, but in this case the LRA
  * that this method is being asked to complete for will be unavailable.
+ * </p>
+ *
+ * <p>
+ * If the method is a JAX-RS resource method (or is a non JAX-RS method
+ * annotated with <code>&#64;Complete</code> with return type
+ * <code>javax.ws.rs.core.Response)</code> then the following are the only
+ * valid response codes:
+ * </p>
+ *
+ *<table border="0" cellpadding="3" cellspacing="0"
+ *   summary="Valid JAX-RS completion response codes">
+ * <caption><span>JAX-RS Completion Response Codes</span><span>&nbsp;</span></caption>
+ * <tr>
+ *   <th scope="col">Code</th>
+ *   <th scope="col">Response Body</th>
+ *   <th scope="col">Meaning</th>
+ * </tr>
+ * <tr>
+ *   <td scope="row">200</td>
+ *   <td scope="row">Empty</td>
+ *   <td scope="row">The resource has successfully completed</td>
+ * </tr>
+ * <tr>
+ *   <td scope="row">202</td>
+ *   <td scope="row">Empty</td>
+ *   <td scope="row">The resource is still attempting completion</td>
+ * </tr>
+ * <tr>
+ *   <td scope="row">404</td>
+ *   <td scope="row">Empty</td>
+ *   <td scope="row">The resource does not know about the LRA</td>
+ * </tr>
+ * <tr>
+ *   <td scope="row">412</td>
+ *   <td scope="row">Empty</td>
+ *   <td scope="row">The resource cannot join the LRA because the LRA is no
+ *   longer active</td>
+ * </tr>
+ * <tr>
+ *   <td scope="row">500</td>
+ *   <td scope="row">{@link ParticipantStatus} enum value</td>
+ *   <td scope="row"><p>The resource has failed to complete.
+ *   The payload contains the reason for the failure.
+ *   A participant MUST remember this state until its
+ *   {@link Forget} method is called.</p>
+ *   <p>The actual value is not important but it MUST
+ *   correspond to a valid {@link ParticipantStatus} enum value. For example,
+ *   if completion was not possible because the resource already
+ *   compensated (without being asked to) then a value such as
+ *   {@link ParticipantStatus#Compensated} would be appropriate or
+ *   if it was due to a generic failure then
+ *   {@link ParticipantStatus#FailedToComplete} would be valid.
+ *   If the response body does not contain a valid status then the
+ *   implementation MUST either reinvoke the method or discover the
+ *   status using the {@link Status} annotation if present.
+ *   </p>
+ *   <p>
+ *   Note that the
+ *   actual state as reported by the {@link Status} method MUST
+ *   be {@link ParticipantStatus#FailedToComplete}</p></td>
+ * </tr>
+ * </table>
+ *
+ * <p>
+ * If any other code is returned (or, in the 500 case, the body does not
+ * correspond to a valid state)  then the implementation MUST either keep
+ * retrying or attempt to discover the status by calling the
+ * {@link Status} method if present or a combination of both.
```

Comment:
also would remove "or a combination of both" unless this is really possible

Reply:
Same response as before